packages/qdrant-loader/tests/unit/config/test_workers_config.py

```
@@ -0,0 +1,77 @@
+        ("1h30m", 5400),
+        ("PT5M", 300),
+        ("PT1H30M", 5400),
+    ],
+)
+def test_parse_interval_seconds_formats(raw, expected):
+    assert parse_interval_seconds(raw) == expected
+
+
+@pytest.mark.parametrize("raw", [0, "", "0s", "PT0S", "abc", True])
+def test_parse_interval_seconds_invalid(raw):
+    with pytest.raises(ValueError):
```

Ruff (0.15.14)

[warning] `pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)